Onboarding qol

NerdRez opened the pull request:
Added features:
- Onboarding tables alternate color/darken row on hover for better readability
- Empty onboarding search result displays "No movies found!" text
- Modal explaining movie poster overlay (desktop + mobile styling) 

Bug fix:
- Fixed movie button logical error

NerdRez committed:
Minor QOL changes

Table alternates colors, empty search gives feedback, updated trigger text

NerdRez committed:
Begin help modal

NerdRez committed:
Table color changes on hover

NerdRez committed:
Merge branch 'master' into onboarding-qol

NerdRez committed:
Fixed onboarding movie list logical bug

Fixed bug where movie would remain in watchlist if user added it to their watchlist and then liked/disliked it.

NerdRez committed:
Help Modal Image + Implementation (Desktop)

Still need to test with varying screen sizes

NerdRez committed:
Movie page help modal implementation (Desktop)

NerdRez committed:
Centered image text

NerdRez committed:
Update image w/ bigger text

Better readability

NerdRez committed:
Mobile modal + bug fix

Fixed bug where back/exit modals too wide and where back/exit modals would not be hidden

NerdRez committed:
Mobile modal styling on movie details page

NerdRez committed:
djlint formatting

NerdRez committed:
Merge branch 'master' into onboarding-qol

jonathanhouge commented on accounts/static/accounts/css/onboarding_base.css:
```diff
@@ -70,6 +70,13 @@ button:hover {
   padding: 0.5rem;
 }
 
+tbody tr:nth-child(odd){
```
let's get your prettier working man

jonathanhouge commented on accounts/static/accounts/css/onboarding_movies.css:
```diff
@@ -111,4 +111,35 @@
+  padding: 1.25rem;
+  color:white;
+  font-size: 1.5rem;
+}
```
no spacing! :0

i am looking for a css linter that we can use, cause apparently prettier doesn't handle this

jonathanhouge commented on landing_page/static/landing_page/images/movie_overlay_instructions.png:
spider-man poster looks cropped (on the right, the 'n' is chipped and it looks close on the left)? can we fix that?

jonathanhouge commented on landing_page/templates/landing_page/movie.html:
```diff
@@ -60,10 +62,19 @@ <h2>{{ movie.name }}</h2>
   // NOTE: ignore IDE error as it is a false positive
   // (fails to interpret Django syntax)
   var movie_data = {{ movie_json | safe}};
+  console.log(movie_data)
```
no 'console.log' in production brother bear

jonathanhouge reviewed (commented):
will review the functionality later today :)

jonathanhouge commented:
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/47656586-b5b9-43fe-b48e-939860d61871)
help question mark looks a little funky

jonathanhouge reviewed (changes requested):
little stuff :)

jonathanhouge commented:
> ![image](https://private-user-images.githubusercontent.com/111781413/344057192-47656586-b5b9-43fe-b48e-939860d61871.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAxNDAxODUsIm5iZiI6MTcyMDEzOTg4NSwicGF0aCI6Ii8xMTE3ODE0MTMvMzQ0MDU3MTkyLTQ3NjU2NTg2LWI1YjktNDNmZS1iNDhlLTkzOTg2MGQ2MTg3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwNVQwMDM4MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NjI1Zjk2M2FkYzJiNjlhMzBkZjM1ZWQzMjZhYzcwMTIwNWJjNDAyMTUyOWViMjU1YzM2OGZjYWI4NTkyMmFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MqGEdwxsPABisC_hXRZfvD4wLniOKyEm_rUwc5MhD4Q) help question mark looks a little funky

![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/6f3fa0ed-f2a1-419f-b099-1bf5abc681b7)
i think this looks better! let me know

jonathanhouge commented:
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/42969f03-4c19-489c-bd2b-d6f70d084d06)
the close button (x) and 'no' on the home page button on the movie onboarding page broke :(
(assuming something to do with the new 'x' being implemented in the help modal)

jonathanhouge committed:
removed console.log & adjusted help buttons

jonathanhouge reviewed (changes requested):
adjust the cropping of 'spider-man' poster in the png & fix the new 'return to home page' bug in 'onboarding_movies.html'

NerdRez committed:
Updated image, fixed modal bugs

Spider-Man poster now full image, fixed mobile styling for close modal buttons on movie page, and fixed home modal

jonathanhouge committed:
deleted old movie_overlay, put it in the proper place

jonathanhouge reviewed (approved):
this is definitely quality of life
